src/server/controllers/receipt.controller.js

```diff
@@ -32,3 +36,15 @@ export default async (req, res) => {
 function isValidPaymentPaymentType(type) {
   return ['FPN', 'CDN', 'IM'].includes(type);
 }
+
+function addFormattedPaymentDateTimes(paymentDetails) {
+  const newPaymentDetails = { ...paymentDetails };
+  const paymentTypes = Object.keys(newPaymentDetails.Payments);
+  for (let i = 0; i < paymentTypes.length; i += 1) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in?